packages/web/app/src/components/ui/page-content-layout.tsx

```diff
@@ -51,14 +51,14 @@ SubPageLayout.displayName = 'SubPageLayout';
 
 type SubPageLayoutHeaderProps = {
   children?: ReactNode;
-  title?: string;
+  subPageTitle?: ReactNode;
```

Comment:
had to rename it as `title` property conflicted with `HTMLDivElement["title"]`, which can only be a string.